### PR TITLE
Capi Traffic Driver Endpoint

### PR DESCRIPTION
**Pull request by Ap0c**
## What does this change?

Adds an endpoint to frontend for retrieving cAPI data about an article by short URL. Used for the gLabs Traffic Driver native template.
- Adds new controller to commercial and wires it up.
- Adds new model to commercial for Traffic Driver data.
- Adds new route to commercial and dev-build for endpoint.
## What is the value of this and can you measure success?

Needed to provide cAPI data for the gLabs Traffic Driver native template, making use of pre-existing code in frontend.
## Request for comment

Not built a component within Play like this before, so would appreciate any comments if e.g. I've done it all horribly wrong @guardian/commercial-dev @janua

### CODE REVIEW

File: commercial/app/controllers/commercial/TrafficDriverController.scala [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    contentApiClient: ContentApiClient,
+    capiAgent: CapiAgent)
+  extends Controller
+  with ExecutionContexts
+  with implicits.Requests
+  with Logging {
+
+    // Request information about the article from cAPI.
+    private def retrieveContent()(implicit request: Request[AnyContent]):
+        Future[Option[ContentType]] = {
```
Comment (janua) [inline commercial/app/controllers/commercial/TrafficDriverController.scala]: Type on a new line?

**Comment by janua** [comment]
:+1:
Maybe you should fix the indentation, it's strange in places!

**Comment by Ap0c** [comment]
Was trying to follow the [style guide](http://docs.scala-lang.org/style/). I've fixed the four-space issue, switched to two.

**Comment by janua** [comment]
:+1:

**Pull request merged**